test/config_parser_SUITE.erl

```diff
@@ -1717,23 +1717,24 @@ mod_extdisco(_Config) ->
     ?errh(T(RequiredOpts#{<<"password">> => <<>>})).
 
 mod_inbox(_Config) ->
+    check_module_defaults(mod_inbox),
+    check_iqdisc_map(mod_inbox),
+    P = [modules, mod_inbox],
     T = fun(Opts) -> #{<<"modules">> => #{<<"mod_inbox">> => Opts}} end,
-    M = fun(Opts) -> modopts(mod_inbox, Opts) end,
     ChatMarkers = [<<"displayed">>, <<"received">>, <<"acknowledged">>],
-    ?cfgh(M([{reset_markers, ChatMarkers}]),
+    ?cfgh(P ++ [reset_markers, ChatMarkers],
```

Comment:
```suggestion
    ?cfgh(P ++ [reset_markers], ChatMarkers,
```
Same applies to the lines below, please see https://github.com/esl/MongooseIM/pull/3548 as well.